src/java.base/share/classes/java/security/SecureClassLoader.java

```diff
@@ -165,16 +165,8 @@ protected final Class<?> defineClass(String name, java.nio.ByteBuffer b,
     }
 
     /**
-     * Returns the permissions for the given CodeSource object.
-     * <p>
-     * This method is invoked by the defineClass method which takes
-     * a CodeSource as an argument when it is constructing the
-     * ProtectionDomain for the class being defined.
-     *
-     * @param codesource the codesource.
-     *
-     * @return the permissions for the codesource.
-     *
```

Comment:
This change will have to go in the CSR as well presumably.

Reply:
Hello Michael, right - I have been talking to Alan and Sean about this change and there has been some inputs from Joe in the CSR too. This latest change to the SecureClassLoader was one proposal that's being considered. I suspect we will need at least one more day to come to a consensus on how we want to proceed. But yes, if we do decide to make this change, I'll update the CSR text.